lib/Twig/Extension/AdminExtension.php

```diff
@@ -28,14 +30,20 @@
  */
 class AdminExtension extends AbstractExtension
 {
-    public function __construct(private UrlGeneratorInterface $generator)
+    private EditmodeResolver $editmodeResolver;
+    private UserLoader $userLoader;
+
+    public function __construct(private UrlGeneratorInterface $generator, EditmodeResolver $editmodeResolver, UserLoader $userLoader)
     {
+        $this->editmodeResolver = $editmodeResolver;
+        $this->userLoader = $userLoader;
     }
```

Comment:
I think we shouldn't mix the syntax.
```suggestion
    public function __construct(
        private UrlGeneratorInterface $generator, 
        private EditmodeResolver $editmodeResolver, 
        private UserLoader $userLoader
    ) {
    }
```